docs/examples/ClasspathExample/README.md

```diff
@@ -12,6 +12,3 @@ How to use the loom.config classpath array to specify source folders to compile.
 
 ## Screenshot
 ![ClasspathExample Screenshot](images/screenshot.png)
-
-## Code
-@insert_source
```

Comment:
Oh. Should we move the source or add an explanatory note in the ## Code section instead of deleting it entirely?